jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/HttpOutput.java

```diff
@@ -174,6 +175,34 @@ public long getWritten()
         return _written;
     }
 
+    /**
+     * Set the content-length as set by the application.  This may not be the actual content length if compression or
+     * similar handlers are used.
+     * @param len The content-length as set by the application.
+     */
+    public void setContentLengthSet(long len)
+    {
+        _contentLengthSet = len;
+    }
+
+    /**
+     * Get the content-length as set by the application.  This may not be the actual content length if compression or
+     * similar handlers are used.
+     * @return The content-length as set by the application.
+     */
+    public long getContentLengthSet()
+    {
+        return _contentLengthSet;
+    }
+
+    /**
+     * @return {@code true} if a Content-Length has been set and insufficient content has been written.
+     */
+    public boolean isContentIncomplete()
+    {
+        return _contentLengthSet >= 0 && _written < _contentLengthSet;
```

Comment:
I'd read the field into a local variable, and use the local twice, rather than the field twice.